eventstore/mongodb_v2/eventstore.go

```diff
@@ -119,6 +121,15 @@ func WithEventHandler(h eh.EventHandler) Option {
 	}
 }
 
+// WithCustomEventsCollectionName uses a different event collection than the default "events".
+func WithCustomEventsCollectionName(collName string) Option {
```

Comment:
Yes, as you stated it should probably set the name of both. Maybe just use to args; eventsColl and streamsColl?

Reply:
Yeah this should do the trick, I'll also introduce an error to check that the function doesn't receive the same collection name for both collections